lib/mockRequest.js

```diff
@@ -76,6 +77,17 @@ function createRequest(options) {
     //parse query string from url to object
     if (Object.keys(mockRequest.query).length === 0) {
         mockRequest.query = require('querystring').parse(mockRequest.url.split('?')[1]);
+
+        if (mockRequest.query.hasOwnProperty) return;
+
+        Object.defineProperty(
+            mockRequest.query,
+            'hasOwnProperty',
+            {
+                enumerable: false,
+                value: Object.hasOwnProperty.bind(mockRequest.query)
+            }
+        );
```

Comment:
Had to do this because for some reason the object returned by `querystring` does not have the `hasOwnProperty` method and was failing the tests.